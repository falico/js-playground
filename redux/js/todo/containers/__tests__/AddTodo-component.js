import React from 'react'
import renderer from 'react-test-renderer'
import AddTodo from '../AddTodo'
import Store from '../../__mocks__/store'

describe('AddTodo', () => {
  it('renders', () => {
    const component = renderer.create(
      <AddTodo store={Store({})} />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})
