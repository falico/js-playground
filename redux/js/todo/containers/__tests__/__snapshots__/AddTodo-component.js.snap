exports[`AddTodo renders 1`] = `
<div>
  <form
    onSubmit={[Function]}>
    <input />
    <button
      type="submit">
      Add Todo
    </button>
  </form>
</div>
`;
